app.py

```diff
@@ -24,6 +24,7 @@ def teardown_logging():
 def add_user(email, password):
     with app.app_context():
         user = User.from_email(db.session, email)
+        email = email.lower()
```

Comment:
this should be the first statement in the function

Reply:
moved it.